src/EnrollTotpController.js

```diff
@@ -111,9 +110,15 @@ const EnrollTotpControllerEnrollTotpController = FormController.extend({
           View: EnrollTotpControllerAppDownloadInstructionsView,
           showWhen: showWhenDeviceTypeSelected,
         }),
-        FormType.Toolbar({
-          noCancelButton: true,
-          save: loc('oform.next', 'login'),
+        FormType.Button({
+          title: loc('oform.next', 'login'),
+          attributes: {
+            'data-type': 'save'
+          },
+          className: 'button button-primary default-custom-button',
+          click: function() {
+            this.model.save();
+          },
```

Comment:
is there a test?

Reply:
Both submit button and newly added button have same attribute and hence no change needed in specs. existing spec works good.